### ISSUE TITLE
create-react-app build fails to minify

### ISSUE THREAD

**Issue by spawnia** [jacobwgillespie/styled-icons#49]
Hi, just today i discovered this project - looks very nice so far.

However, i have the following issue when trying to create the production build with the `create-react-app` build step:

````
$ yarn build
yarn run v1.6.0
(node:49) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
$ react-scripts build
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/styled-icons/fa-brands/FiveHundredPx.js:4 

Read more here: http://bit.ly/2tRViJ9
````

**Comment by jacobwgillespie** [comment]
Hey, thanks for the report - I’d be happy to take a look.  Can you provide the following, if you are able:

* What version of NodeJS are you using?
* What does your `package.json` contain (especially the version of react-scripts)?
* Can you post an example of how you are importing styled-icons?

**Comment by spawnia** [comment]
What version of NodeJS are you using?

I am running in the docker image `node:alpine`.

````
docker run -it -v ${PWD}:/workdir node:alpine sh
/ # node --version
v10.0.0
````

You can actually take a look at my repo here: (make sure to stick with that revision, unfortunately i kicked styled-icons after this)

https://gitlab.com/spawnia/franke.tech/tree/c8cccd6b421b2b4c24325b805728fca08f389633

I am using the icons in here:

https://gitlab.com/spawnia/franke.tech/blob/c8cccd6b421b2b4c24325b805728fca08f389633/src/pages/index.js

**Comment by jacobwgillespie** [comment]
Thanks again for the report and the link to your repo, I was able to track down the issue.  It appears that I had not properly tested this repo against Create React App, as the app I tested it with was modifying UglifyJS's minification settings.

I just released v2.0.0 which fixes that issue.  I opened a merge request against your repo to demonstrate using the updated version: https://gitlab.com/spawnia/franke.tech/merge_requests/1.
